plato: I think I figured it out... combination of a bad python_home and a missing gcc-c++ install.